### PR TITLE
adding bruceandmorgan.com scraper

### PR DESCRIPTION
**Pull request by Tweeticoats**
adding bruceandmorgan.com scraper

### CODE REVIEW

File: scrapers/bruceandmorgan.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        selector: //div[@class='title']/h2/span/text()
+      Details: //h3[@class='full-desc']/span/text()
+      Tags:
+        Name: //h4[@class='tags content-info']/span/a/span/text()
+      Studio:
+        Name:
+          fixed: Bruce and Morgan
+driver:
+  useCDP: true
+# Last Updated September 30, 2020
```
Comment (halorrr) [inline scrapers/bruceandmorgan.yml:20]: Date needs updating

File: scrapers/bruceandmorgan.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+name: Bruce and Morgan
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://bruceandmorgan.net/scene/
```
Comment (halorrr) [inline scrapers/bruceandmorgan.yml:5]: Protocol should be dropped from the URL so that it is just `bruceandmorgan.net/scene/`

**Comment by halorrr** [comment]
Based on my comments this would be the updated version:

```YAML
name: Bruce and Morgan
sceneByURL:
  - action: scrapeXPath
    url:
      - bruceandmorgan.net/scene/
    scraper: sceneScraper
xPathScrapers:
  sceneScraper:
    scene:
      Title:
        selector: //div[@class='title']/h2/span/text()
      Details: //h3[@class='full-desc']/span/text()
      Tags:
        Name: //h4[@class='tags content-info']/span/a/span/text()
      Studio:
        Name:
          fixed: Bruce and Morgan
driver:
  useCDP: true

# Last Updated October 23, 2020
```

File: SCENE-SCRAPABLE.md [hunk 3]
```diff
@@ -98,6 +98,7 @@ boxtrucksex.com|BoxTruckSex.yml|
 boysdestroyed.com|AMAMultimedia.yml|Gay
 brattysis.com|Nubiles.yml|
 brazzers.com|Brazzers.yml|
+bruceandmorgan.com|bruceandmorgan.yml|Amateur Fetish
```
Comment (SpedNSFW) [inline SCENE-SCRAPABLE.md:101]: You're scraping bruceandmorgan.net, not .com.

```suggestion
bruceandmorgan.net|bruceandmorgan.yml|Amateur Fetish
```

**Pull request merged**